.github/workflows/Image.yaml

```diff
@@ -0,0 +1,60 @@
+
+name: Images build series for Environment  
+
+on: 
+  push:
+    branches:
+      - 'master'
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker login
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USER }}   
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}  
+      run: |
+        docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+
+    - name: docker build binary 
+      run: |
+        docker build ./binary -t fission/binary:binary_latest
```

Comment:
The image tag should be just `latest` . Also, it would be nice if we had some versioning in place like we have in this PR. https://github.com/fission/keda-connectors/pull/61/files